intro-Cloud/ch_Flavors.tex

```diff
@@ -0,0 +1,96 @@
+\chapter{Instance types and flavors}
+
+VSC Tier1 Cloud provides several virtual machine instance types
+and flavors to fit different use cases.
+Each instance type provides several flavor sizes to give different
+combinations of CPU, memory, GPU and network resource.
+
+\section{Instance Types}\label{sec:instance-types}
+The following table provides the current main instance types available
+from VSC Tier1 Cloud infrastructure:
+
+\begin{table}[h!]
+\centering
+\begin{tabular}{ |p{6cm}|p{4cm}|p{5cm}| }
+  \hline
+  \rowcolor{lightgray} \textbf{UPSv1} & \textbf{CPUv1} & \textbf{GPUv1} \\
+  \hline
+  \begin{itemize}
+    \item AMD Epyc 7542 2.9GHz
+    \item vCPU oversubscription 2:1
+    \item 25Gbit Ethernet
+    \item Uninterruptible Power Supply (UPS)
+  \end{itemize}
+  &
+  \begin{itemize}
+    \item Intel Xeon CPU E5-2670 2.60GHz
+    \item 10Gbit Ethernet
+  \end{itemize}
+  &
+  \begin{itemize}
+    \item AMD Epyc 7542 2.9GHz
+    \item 25Gbit Ethernet
+    \item NVIDIA Tesla 4 GPU
+  \end{itemize}
+  \\
+  \hline
+\end{tabular}
+\caption{Instance types hardware profiles}
+\label{table:instance-type}
+\end{table}
+
+Each instance type is appropriate for different workloads. From regular 
+CPU usage (\strong{CPUv}\emph{<version>} instance types), to high performance 
+GPU computations (\strong{GPUv}\emph{<version>}) or VMs connected to 
+uninterruptible power supplies (\strong{UPSv}\emph{<version>}). VMs using UPS 
+flavor types will be keep up and running even if the data centre suffers
+an unexpected power cut.
+
+VSC Tier-1 Cloud instance types also provide different kind of network
+performance specifications. All the instance types are able to connect
+to the available networks: public network, VSC network and shared filesystem
+network (NFS). VSC and shared file system networks are only available
```

Comment:
perhaps make it clear that the vsc and nfs networks give an optimal path towards the other vsc services or the nfs exports; it does not imply e.g. direct access to all services in case of VSC

Reply:
@stdweird ok I have included more info.